lib/gslab_model/ModelEstimationOutput.R

```diff
@@ -0,0 +1,8 @@
+ModelEstimationOutput <- setRefClass(Class  = "ModelEstimationOutput",
```

Comment:
I'm not sure whether we want to add a default constructor. We want to [allow](https://github.com/gslab-econ/gslab_r/issues/2#issuecomment-307434088) different optimization methods, so the default constructor is very likely to be overrode. What do you think?